ui.spec

```diff
@@ -1,116 +1,59 @@
-%if %{?rhel}%{!?rhel:0} >= 7
-%global el6 0
-%else
-%global el6 1
-%endif
-Name:		ui
-Version:	4.0.3
-Release:	1%{?dist}
-Summary:	UI meta-packages
-Group:		Applications/Internet
-License:	ASL 2.0
-BuildRoot:	%(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
+Name: ui
+Version: 4.0.3
+Release: 1%{?dist}
+Summary: UI meta-packages
+Group: Applications/Internet
+License: ASL 2.0
+BuildRoot: %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
 
 # the above replaced by ca-policy-egi-core
-Requires:       ca-policy-egi-core
-Requires:       aria2
-Requires:	      davix-libs
-Requires:       dcache-srmclient
-Requires:       dcap
-Requires:	      dcap-devel
-Requires:	      dcap-libs 
-Requires:	      dcap-tunnel-gsi
-Requires:	      dcap-tunnel-krb  
-Requires:	      dcap-tunnel-ssl  
-Requires:	      dcap-tunnel-telnet
-Requires:       dpm
-Requires:       dpm-devel
-Requires:       dpm-perl
-Requires:       dpm-python
-Requires:       lcgdm-devel
-Requires:       libdpm.so.1()(64bit), dpm-libs
-Requires:       liblcgdm.so.1()(64bit), lcgdm-libs
-Requires:	      fetch-crl
-Requires:	      fts-client
-Requires:     	 fuse
-Requires:       fuse-libs
-Requires:       gfal2-all
-Requires:	      gfal2-plugin-xrootd
-Requires:       gfal2-python
-Requires:	      gfal2-util
-Requires:       gfalFS
-Requires:	      gfal2-doc
-Requires:	      gfal2-devel
-Requires:	      ginfo
-Requires:       glite-ce-cream-cli  
-Requires:       glite-ce-cream-client-api-c
-Requires:	      glite-yaim-core
-Requires:	      glite-yaim-clients
-Requires:	      gsi-openssh-clients
-Requires:	      globus-gsi-cert-utils-progs
-Requires:       lcg-ManageVOTag  
-Requires:       lcg-info  
-Requires:       lcg-infosites  
-Requires:       lcg-tags  
-Requires:       lfc
-Requires:       liblfc.so.1()(64bit), lfc-libs
-Requires:       lfc-devel
-Requires:       lfc-perl
-Requires:       lfc-python
-Requires:	      myproxy
-Requires:       nordugrid-arc-client
-Requires:       nordugrid-arc-plugins-xrootd
-Requires:       nordugrid-arc-plugins-gfal
-Requires:	      nordugrid-arc-plugins-globus
-Requires:	      nordugrid-arc-plugins-needed
-Requires:       openldap-clients
-Requires:       voms  
-Requires:       voms-clients-java  
-Requires:	      xrootd-client
+Requires: ca-policy-egi-core
+Requires: aria2
+Requires: condor
+Requires: davix-libs
+Requires: dcache-srmclient
+Requires: dcap
+Requires: dcap-devel
+Requires: dcap-libs
+Requires: dcap-tunnel-gsi
+Requires: dcap-tunnel-krb
+Requires: dcap-tunnel-ssl
+Requires: dcap-tunnel-telnet
+Requires: fetch-crl
+Requires: fts-client
+Requires: fuse
+Requires: fuse-libs
+Requires: gfal2-all
+Requires: gfal2-plugin-xrootd
+Requires: gfal2-python
+Requires: gfal2-util
+Requires: gfalFS
+Requires: gfal2-doc
+Requires: gfal2-devel
+Requires: ginfo
+Requires: glite-yaim-core
```

Comment:
is yaim still used somewhere?

Reply:
I would tend to say no, and even if it may still be the case for some components (didn't installed a site since so long that I've no idea what would actually still be required or not in certain cases, and as we don't really have any up to date documentation for this...), I'm not sure it would make sense to have this on a UI.
@paolini78 what do you think?

Reply:
on UI yaim was only setting the env variables and the files for the VOs (.lsc and vomses) so maybe it can be dropped, and in the docs it can be explained how to configure the VOs and the needed env variables

Reply:
OK. 
In fact I'm not aware of any existing documentation for the UI, so not sure where we should document it.
We could at least add it to the repo README.md and ideally to docs.egi.eu too.

For setting up the UI, the only thing I tried (and fixed) is using the ansible roles [UI](https://github.com/EGI-Federation/ansible-role-ui) and [VOMS-client](https://github.com/EGI-Federation/ansible-role-VOMS-client) roles, with the VOMS one doing all the work: https://github.com/EGI-Federation/ansible-role-VOMS-client/blob/main/tasks/cache_config.yml#L26-L46
Have you a pointer to some old doc regarding how to create those files? If not I will draft something based on what the voms-client role does with the info from the ops portal.

Reply:
on voms-admin page like https://voms2.hellasgrid.gr:8443/voms/dteam/configuration/configuration.action;jsessionid=qwp8cif6mf5k1tekwum6tv2uy you find the content to put in those files but the location is not reported. So you just need to report the location.
On the UI you can locate "vomses" and ".lsc" to find the exact place.
Also in this pages https://wiki.egi.eu/wiki/Dteam_vo and https://wiki.egi.eu/wiki/OPS_vo the info is not reported.

Reply:
the old YAIM guide https://twiki.cern.ch/twiki/bin/view/LCG/YaimGuide400

Reply:
i'm aware that cern is also hosting rpms to configure each VO easily https://dmc-repo.web.cern.ch/dmc-repo/el7/x86_64/ ( see voms-config-vo* rpms)

Reply:
So in fact I re-used/updated what I added the HTC tutorial, I had forgotten that I already had documented this :)

Reply:
Interesting @andrea-manzi but it's not something we can reference/use, and it's mostly for the LHC VOs.